lib/rooms.rb

```diff
@@ -0,0 +1,28 @@
+# Room logic.  SIMPLE
+# I need this to provide me with 20 numbered rooms starting at the number 1 and ending at the number 20.
+# This also needs to (I believe) hold the room cost.  Each room has a $200 charge per night.
+
+module HotelHedwig
+  class Rooms
```

Comment:
I would probably name this class `Room` instead of `Rooms`.  Since it describes a single room.